docs/embedding.md

```diff
@@ -0,0 +1,62 @@
+---
+sidebar_position: 7
+---
+
+# Embedding Apex into Apps
```

Comment:
Can you give any examples of why anyone would want to do this?  This should not be a commonly used workflow.

Reply:
Yep, good idea.